tests/pos-java-interop/i7212/UseFromJava.java

```diff
@@ -0,0 +1,8 @@
+public class UseFromJava {
```

Comment:
I recommend turning this into a run test (they go in tests/run and need a class Test with a static main method in them) to ensure that we're emitting valid bytecode.

Reply:
moved to tests/run/i7212